src/main/java/uk/gov/justice/hmpps/prison/security/AuthenticationFacade.java

```diff
@@ -1,95 +0,0 @@
-package uk.gov.justice.hmpps.prison.security;
-
-import lombok.extern.slf4j.Slf4j;
-import org.apache.commons.lang3.RegExUtils;
-import org.springframework.security.authentication.AbstractAuthenticationToken;
-import org.springframework.security.core.Authentication;
-import org.springframework.security.core.GrantedAuthority;
-import org.springframework.security.core.context.SecurityContextHolder;
-import org.springframework.security.core.userdetails.UserDetails;
-import org.springframework.stereotype.Component;
-import uk.gov.justice.hmpps.kotlin.auth.AuthAwareAuthenticationToken;
-import uk.gov.justice.hmpps.kotlin.auth.AuthSource;
-
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-
-@Component
-@Slf4j
-public class AuthenticationFacade {
-
-    public Authentication getAuthentication() {
-        return SecurityContextHolder.getContext().getAuthentication();
-    }
-
-    public String getCurrentPrincipal() {
-        final String username;
-
-        final var userPrincipal = getUserPrincipal();
-
-        if (userPrincipal instanceof String) {
```

Comment:
not previous implementation could't tell what type of principal was being returned - test code has all refactored to now use the WithMockAuthUser so we know it will be a string